packages/headless/src/api/search/search-api-client.ts

```diff
@@ -1,3 +1,4 @@
+import AbortController from 'node-abort-controller';
```

Comment:
This is not supported out of the box by nodejs ? Bummer...

What impact does it have for browser environments ? 
Can we arrange for it to be imported only when needed ?

Reply:
The goal of this package is to use the browser's AbortController when available and provide a small polyfill when on node

https://www.npmjs.com/package/node-abort-controller